File: library/src/main/java/com/devbrackets/android/androidmarkup/parser/HtmlParser.java
package com.devbrackets.android.androidmarkup.parser;

import android.text.Html;
import android.text.Spanned;

/**
 * A MarkupParser for the Html protocol
 */
public class HtmlParser extends MarkupParser {
    @Override
    public Spanned toSpanned(String text) {
        return Html.fromHtml(text);
    }

    @Override
    public String fromSpanned(Spanned spanned) {
        return Html.toHtml(spanned);
    }
}

File: library/src/main/java/com/devbrackets/android/androidmarkup/parser/HtmlParser.kt
package com.devbrackets.android.androidmarkup.parser

import android.text.Html
import android.text.Spanned

/**
 * A MarkupParser for the Html protocol
 */
class HtmlParser : MarkupParser() {
    override fun toSpanned(text: String): Spanned {
        return Html.fromHtml(text)
    }

    override fun fromSpanned(spanned: Spanned): String {
        return Html.toHtml(spanned)
    }
}
